corehq/apps/domain/deletion.py

```diff
@@ -413,6 +413,7 @@ def _delete_demo_user_restores(domain_name):
     ModelDeletion('couchforms', 'UnfinishedSubmissionStub', 'domain'),
     ModelDeletion('couchforms', 'UnfinishedArchiveStub', 'domain'),
     CustomDeletion('ucr', delete_all_ucr_tables_for_domain, []),
+    ModelDeletion('domain', 'OperatorCallLimitSettings', 'domain'),
```

Comment:
This should also be added to the [SQL dump code](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/dump_reload/sql/dump.py).

Reply:
ok, strangely no test failed for it, similar to how it failed for not adding this deletion entry, which made me aware of this file in the first place.
Will add.

Reply:
that is an excellent point, this would be good to have a test for

Reply:
There seem to be many missing models in the sql dump file. Not sure if this should be taken as a separate activity to collect what models are actually needed to be put here.

Reply:
Probably, though I'm unclear on who owns that code. Might be a good topic for cross-divisional dev leads.

Reply:
Good idea.